syntax = "proto3";

package vendas;

service PassagensService {
  rpc VerificarDisponibilidade (PassagemRequest) returns (PassagemResponse) {}
}

message PassagemRequest {
  string data_ida = 1;
  string data_volta = 2;
  string origem = 3;
  string destino = 4;
}

message PassagemResponse {
  bool disponibilidade = 1;
  string categoria = 2;
}
